frameworks/cassandra/build.sh

```diff
@@ -9,7 +39,12 @@ ROOT_DIR="$(dirname "$(dirname $FRAMEWORK_DIR)")"
 export TOOLS_DIR=${ROOT_DIR}/tools
 BUILD_DIR=$FRAMEWORK_DIR/build/distributions
 PUBLISH_STEP=${1-none}
-${ROOT_DIR}/gradlew distZip -p ${ROOT_DIR}/sdk/executor
+
+if [ x"$cli_only" = xtrue ]; then
```

Comment:
@mellenburg this approach is a pretty common pattern with shell programming. I tend to prefer using the modern bash [double-brack tests](http://tldp.org/LDP/abs/html/comparison-ops.html) which allows for more powerful and readable expressions, but the `x"$cli_only"...` test is also fine IMO.